project/npda/templates/partials/audit_upload_history.html

```diff
@@ -0,0 +1,66 @@
+{% load static %}
+{% if auditcohorts %}
+            <table class="table table-md w-full text-sm text-left rtl:text-right text-gray-500 text-gray-400 mb-5 font-montserrat">
+                <thead class="text-xs text-gray-700 uppercase bg-gray-50 bg-rcpch_dark_blue text-white">
+                    <tr>
+                        <th>Submission Date</th>
+                        <th>Submission By</th>
+                        <th>Audit Year (Cohort)</th>
+                        <th>Patient Number</th>
+                        <th>Active</th>
+                        <th>Download</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for audit in auditcohorts %}
+                    {% if forloop.first %}
+                    <tr class="border-b bg-white text-rcpch_light_blue border-gray-100 hover:bg-r hover:bg-rcpch_light_blue hover:text-white">
+                    {% else %}
+                    <tr class="border-b bg-rcpch_lightest_grey text-rcpch_light_blue border-gray-100 hover:bg-r hover:bg-rcpch_mid_grey hover:text-white">
+                    {% endif %}
+                            <td>{{ audit.submission_date|date:"d/m/Y H:i s" }} secs</td>
+                            <td>{{audit.submission_by}}</td>
+                            <td>{{audit.audit_year}} ({{audit.cohort_number}})</td>
+                            <td>{{audit.patient_count}}</td>
+                            <td>{{audit.submission_active}}</td>
+                            <td class="flex flex-row">
+                                <form method="post" action="{% url 'audit_cohorts' %}" class="join">
+                                    {% csrf_token %}
+                                    <input type="hidden" name="audit_id" value="{{audit.pk}}">
+                                    <button name="submit-data" value="download-data" type="submit" class="join-item bg-rcpch_light_blue text-white font-semibold hover:text-white py-1 px-2 border border-rcpch_light_blue hover:bg-rcpch_strong_blue hover:border-rcpch_strong_blue btn-sm rounded-none">Download</button>
```

Comment:
Just signposting that this button doesn't yet have functionality. Delete button below works fine though.

Reply:
do please raise a separate issue for this if you have time